src/Hipchat.php

```diff
@@ -43,7 +43,10 @@ function postOutput($outputString)
             echo $outputString;
         } else {
             $outputString = $this->convertNewLineToBr($outputString);
-            $this->postInHipChat($outputString);
+            if (!$this->wasLastMessageTheSameAsThisMessage($outputString))
```

Comment:
that's fair, I was thinking basically checking to see if the room is really active before talking so it works at any time of day/week

I was also thinking with this we can or should go back to every 5 minutes instead of 15